compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1807,6 +1807,62 @@ CustomOp::reifyResultShapes(OpBuilder &builder,
   return success();
 }
 
+/// `AggregateOpInterface` implementation
```

Comment:
This comment is meaningless to me, perhaps we can drop it?

Reply:
I wanted to demarcate all the interface implementations. Did that better now and moved a method that should have been moved with the tiling interface implementation.